### ISSUE TITLE
Complete dry run of all Shamir shares

### ISSUE THREAD

**Issue by ZdenekSL** [trezor/trezor-firmware#276]
Possibility to run dry run of all Shamir shares. User in the wallet should be asked, after threshold, if he wants to check all remaining shares for validity.

**Comment by ciny** [comment]
WIP on this is in ciny/shamir_dryrun branch. UI and workflow should work. Based on discussion at standup we'll finish this up after #270 is done.

**Comment by tsusanka** [comment]
@ZdenekSL didn't we decide that we won't do this unless Product decides so?

**Comment by ZdenekSL** [comment]
@tsusanka yup, that's why there is product owner label :)

**Comment by ZdenekSL** [comment]
Product Owner decided that we will implement this in the future.

**Comment by sorooris** [comment]
FYI this issue is also present in the Shamir groups - user is prompted to enter another group shares after an in-group share threshold is reached.
